source/macros.tex

```diff
@@ -198,7 +198,7 @@
 \newcommand{\leftshift}[1]{\ensuremath{\, \mathsf{lshift}_#1 \,}}
 
 %% Notes and examples
-\newcommand{\noteintro}[1]{[\,\textit{#1:}\space}
+\newcommand{\noteintro}[1]{[\,\textit{#1}:\space}
```

Comment:
Yes, some, but less. I can show you some variations at some point. But let's separate that from the bnf-colon issue.